_i18n/en.yml

```diff
@@ -794,9 +794,17 @@ help_pages:
       a new security code. Each security code is only good once, so no one can steal
       one you've already used. We limit requests to three every 15 minutes, but otherwise
       you can request as many codes as you need.
-    what-is-an-authenticator-app: A mobile security application that generates security
-      codes for signing in. You can use the app to get codes even if you don't have
-      an internet connection or mobile service.
+    what-is-an-authentication-app: |-
+      Authentication apps generate security codes for signing in to sites that require a high level of security. You can use these apps to get security codes even if you don’t have an internet connection or mobile service.
+
+      You can set up an authentication app to generate your login.gov security code. First, you’ll need to download an authentication app to your computer or phone. After installing and configuring the application to work with login.gov, you will be able to receive security codes without a phone number. Some options for authentication apps include:
+
+        - Android options: [Google Authenticator](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&hl=en){:target="_blank"}, [Authy](https://authy.com){:target="_blank"}, [LastPass](https://lastpass.com){:target="_blank"}, [1Password](https://1password.com){:target="_blank"}
+        - iOS options: [Google Authenticator](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&hl=en){:target="_blank"}, [Authy](https://authy.com){:target="_blank"}, [LastPass](https://lastpass.com){:target="_blank"}, [1Password](https://1password.com){:target="_blank"}
```

Comment:
The Google Authenticator link for the iOS option is pointing to the Android Play store, not the app on the Apple Store.

Reply:
Bah! Good catch. I also saw another on for the Windows OTP app. I've corrected those and pushed.